pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   logging: ^1.2.0
   protobuf: ^3.1.0
   string_validator: ^1.0.2
-  uuid: ^4.3.1
+  uuid: ^4.2.2
```

Comment:
flutter demo app couldn't import the momento package because flutter uses uuid < 4.3.0

Reply:
hmmmm, makes me question how dart handles transient deps, and if our other deps could cause a similar issue with other programs. I think is OK for now, but something to keep an eye out on